viewSQLposts.php

```diff
@@ -0,0 +1,37 @@
+<?php
+include ('include/SQLProject.php');
+?>
+
+
+<html>
+	<head>
+		<title> View Blog Posts </title>
+		<link rel='stylesheet' href='maindesign.css' />
+	</head>
+</html>
+
+<?php
+include('config/config.php');
+include('include/db_query.php');
+
+
+function GetAllBlogposts($blogpostId){
```

Comment:
I'd make a file for blog-related CRUD functions in your include folder. This function should go in there. As you build out the blog, you'll have a lot more functions like inserting new blog posts, getting all of the posts at once, searching posts, inserting comments, getting comments, etc.

The main reason you want it in an include file instead of here is that you might want to use this function on other pages.